compiler/modules/types/core.bal

```diff
@@ -993,6 +1007,15 @@ public function containsConstBoolean(SemType t, boolean b) returns boolean {
     }
 }
 
+public function containsConstDecimal(SemType t, decimal d) returns boolean {
+    if t is UniformTypeBitSet {
+        return (t & (1 << UT_DECIMAL)) != 0;
+    }
+    else {
+        return decimalSubtypeContains(getComplexSubtypeData(t, UT_DECIMAL), d);
+    }
```

Comment:
`else` is not needed.

Reply:
It may be required due to https://github.com/ballerina-platform/ballerina-spec/issues/73

Reply:
I am sorry my comment is misleading. What I want to say is we can do the 2nd return without else block.

Reply:
We need to explicitly cast `t` without the else block